### ISSUE TITLE
Deprecation warnings on the current 1.13.x beta

### ISSUE THREAD

**Issue by dflock** [simurai/duotone-dark-syntax#25]
Here's what deprecation cop says:

Starting from Atom v1.13.0, the contents of atom-text-editor elements are no longer encapsulated within a shadow DOM boundary. This means you should stop using :host and ::shadow pseudo-selectors, and prepend all your syntax selectors with syntax--. To prevent breakage with existing style sheets, Atom will automatically upgrade the following selectors:

```
atom-text-editor, :host => atom-text-editor,atom-text-editor
atom-text-editor .line.cursor-line, :host .line.cursor-line => atom-text-editor .line.cursor-line,atom-text-editor .line.cursor-line
atom-text-editor .invisible, :host .invisible => atom-text-editor .invisible,atom-text-editor .invisible
atom-text-editor .cursor, :host .cursor => atom-text-editor .cursor,atom-text-editor .cursor
atom-text-editor .selection .region, :host .selection .region => atom-text-editor .selection .region,atom-text-editor .selection .region
atom-text-editor .bracket-matcher .region, :host .bracket-matcher .region => atom-text-editor .bracket-matcher .region,atom-text-editor .bracket-matcher .region
atom-text-editor .invisible-character, :host .invisible-character => atom-text-editor .invisible-character,atom-text-editor .invisible-character
atom-text-editor .indent-guide, :host .indent-guide => atom-text-editor .indent-guide,atom-text-editor .indent-guide
atom-text-editor .wrap-guide, :host .wrap-guide => atom-text-editor .wrap-guide,atom-text-editor .wrap-guide
atom-text-editor .find-result .region.region.region, :host .find-result .region.region.region, atom-text-editor .current-result .region.region.region, :host .current-result .region.region.region => atom-text-editor .find-result .region.region.region,atom-text-editor .find-result .region.region.region, atom-text-editor .current-result .region.region.region,atom-text-editor .current-result .region.region.region
atom-text-editor .find-result .region.region.region, :host .find-result .region.region.region => atom-text-editor .find-result .region.region.region,atom-text-editor .find-result .region.region.region
atom-text-editor .current-result .region.region.region, :host .current-result .region.region.region => atom-text-editor .current-result .region.region.region,atom-text-editor .current-result .region.region.region
atom-text-editor .gutter .line-number, :host .gutter .line-number => atom-text-editor .gutter .line-number,atom-text-editor .gutter .line-number
atom-text-editor .gutter .line-number.git-line-removed:before, :host .gutter .line-number.git-line-removed:before => atom-text-editor .gutter .line-number.git-line-removed:before,atom-text-editor .gutter .line-number.git-line-removed:before
atom-text-editor .gutter .line-number.git-line-removed:after, :host .gutter .line-number.git-line-removed:after => atom-text-editor .gutter .line-number.git-line-removed:after,atom-text-editor .gutter .line-number.git-line-removed:after
atom-text-editor .gutter .line-number.cursor-line, :host .gutter .line-number.cursor-line => atom-text-editor .gutter .line-number.cursor-line,atom-text-editor .gutter .line-number.cursor-line
atom-text-editor .gutter .line-number.cursor-line-no-selection, :host .gutter .line-number.cursor-line-no-selection => atom-text-editor .gutter .line-number.cursor-line-no-selection,atom-text-editor .gutter .line-number.cursor-line-no-selection
atom-text-editor .gutter .line-number .icon-right, :host .gutter .line-number .icon-right => atom-text-editor .gutter .line-number .icon-right,atom-text-editor .gutter .line-number .icon-right
atom-text-editor .gutter .line-number.folded, :host .gutter .line-number.folded, atom-text-editor .gutter .line-number:after, :host .gutter .line-number:after, atom-text-editor .fold-marker:after, :host .fold-marker:after => atom-text-editor .gutter .line-number.folded,atom-text-editor .gutter .line-number.folded, atom-text-editor .gutter .line-number:after,atom-text-editor .gutter .line-number:after, atom-text-editor .fold-marker:after,atom-text-editor .fold-marker:after
.html.elements, .entity, .tag, .function-call => .syntax--html.syntax--elements, .syntax--entity, .syntax--tag, .syntax--function-call
.attribute-name, .character.escape => .syntax--attribute-name, .syntax--character.syntax--escape
.support => .syntax--support
.variable => .syntax--variable
.string, .constant, .storage.type => .syntax--string, .syntax--constant, .syntax--storage.syntax--type
.keyword, .storage => .syntax--keyword, .syntax--storage
.punctuation, .bracket, .brace => .syntax--punctuation, .syntax--bracket, .syntax--brace
.string .punctuation => .syntax--string .syntax--punctuation
.comment => .syntax--comment
.bold => .syntax--bold
.italic => .syntax--italic
.c .parens, .c .block => .syntax--c .syntax--parens, .syntax--c .syntax--block
.c .parens .punctuation, .c .block .punctuation, .c .parens .bracket, .c .block .bracket, .c .parens .brace, .c .block .brace => .syntax--c .syntax--parens .syntax--punctuation, .syntax--c .syntax--block .syntax--punctuation, .syntax--c .syntax--parens .syntax--bracket, .syntax--c .syntax--block .syntax--bracket, .syntax--c .syntax--parens .syntax--brace, .syntax--c .syntax--block .syntax--brace
.c .parens .string .punctuation, .c .block .string .punctuation => .syntax--c .syntax--parens .syntax--string .syntax--punctuation, .syntax--c .syntax--block .syntax--string .syntax--punctuation
.c .comment => .syntax--c .syntax--comment
.source.clojure .expression => .syntax--source.syntax--clojure .syntax--expression
.source.clojure .expression .punctuation => .syntax--source.syntax--clojure .syntax--expression .syntax--punctuation
.source.clojure .symbol => .syntax--source.syntax--clojure .syntax--symbol
.source.clojure .vector => .syntax--source.syntax--clojure .syntax--vector
.coffee.source => .syntax--coffee.syntax--source
.source.cs .meta => .syntax--source.syntax--cs .syntax--meta
.source.cs .method => .syntax--source.syntax--cs .syntax--method
.css.source => .syntax--css.syntax--source
.css.support.property-name => .syntax--css.syntax--support.syntax--property-name
.css.attribute-name.id, .css.attribute-name.class, .css.attribute-name.parent-selector => .syntax--css.syntax--attribute-name.syntax--id, .syntax--css.syntax--attribute-name.syntax--class, .syntax--css.syntax--attribute-name.syntax--parent-selector
.css.unit => .syntax--css.syntax--unit
.css.function => .syntax--css.syntax--function
.css.punctuation.terminator => .syntax--css.syntax--punctuation.syntax--terminator
.css.at-rule => .syntax--css.syntax--at-rule
.css.at-rule .keyword.punctuation => .syntax--css.syntax--at-rule .syntax--keyword.syntax--punctuation
.gfm => .syntax--gfm
.gfm .punctuation, .gfm .bracket, .gfm .brace => .syntax--gfm .syntax--punctuation, .syntax--gfm .syntax--bracket, .syntax--gfm .syntax--brace
.gfm .string .punctuation => .syntax--gfm .syntax--string .syntax--punctuation
.gfm .link => .syntax--gfm .syntax--link
.gfm.markup => .syntax--gfm.syntax--markup
.gfm.markup.bold => .syntax--gfm.syntax--markup.syntax--bold
.gfm.markup.italic => .syntax--gfm.syntax--markup.syntax--italic
.gfm.markup.strike => .syntax--gfm.syntax--markup.syntax--strike
.gfm.markup.raw => .syntax--gfm.syntax--markup.syntax--raw
.gfm.markup.raw .support, .gfm.markup.code .support => .syntax--gfm.syntax--markup.syntax--raw .syntax--support, .syntax--gfm.syntax--markup.syntax--code .syntax--support
.gfm.comment => .syntax--gfm.syntax--comment
.gfm.comment.quote => .syntax--gfm.syntax--comment.syntax--quote
.gfm.table => .syntax--gfm.syntax--table
.gfm.table .border, .gfm.table .pipe => .syntax--gfm.syntax--table .syntax--border, .syntax--gfm.syntax--table .syntax--pipe
.gfm.variable, .gfm.entity => .syntax--gfm.syntax--variable, .syntax--gfm.syntax--entity
.gfm.support => .syntax--gfm.syntax--support
.gfm.heading-1, .gfm.heading-2, .gfm.heading-3 => .syntax--gfm.syntax--heading-1, .syntax--gfm.syntax--heading-2, .syntax--gfm.syntax--heading-3
.gfm.heading-4, .gfm.heading-5, .gfm.heading-6 => .syntax--gfm.syntax--heading-4, .syntax--gfm.syntax--heading-5, .syntax--gfm.syntax--heading-6
.haskell.source => .syntax--haskell.syntax--source
.haskell.source .punctuation, .haskell.source .bracket, .haskell.source .brace => .syntax--haskell.syntax--source .syntax--punctuation, .syntax--haskell.syntax--source .syntax--bracket, .syntax--haskell.syntax--source .syntax--brace
.haskell.source .string .punctuation => .syntax--haskell.syntax--source .syntax--string .syntax--punctuation
.html.meta => .syntax--html.syntax--meta
.html.embedded .meta, .html.doctype, .html.text => .syntax--html.syntax--embedded .syntax--meta, .syntax--html.syntax--doctype, .syntax--html.syntax--text
.html .string .id => .syntax--html .syntax--string .syntax--id
.jade.text, .jade .constant.name.attribute.tag => .syntax--jade.syntax--text, .syntax--jade .syntax--constant.syntax--name.syntax--attribute.syntax--tag
.jade .meta.control.flow => .syntax--jade .syntax--meta.syntax--control.syntax--flow
.jade .attribute-name.id => .syntax--jade .syntax--attribute-name.syntax--id
.java .method => .syntax--java .syntax--method
.java .method .punctuation, .java .method .bracket, .java .method .brace => .syntax--java .syntax--method .syntax--punctuation, .syntax--java .syntax--method .syntax--bracket, .syntax--java .syntax--method .syntax--brace
.java .method .string .punctuation => .syntax--java .syntax--method .syntax--string .syntax--punctuation
.java .dereference => .syntax--java .syntax--dereference
.js.delimiter => .syntax--js.syntax--delimiter
.js.source => .syntax--js.syntax--source
.js.source .punctuation, .js.source .bracket, .js.source .brace => .syntax--js.syntax--source .syntax--punctuation, .syntax--js.syntax--source .syntax--bracket, .syntax--js.syntax--source .syntax--brace
.js.source .string .punctuation => .syntax--js.syntax--source .syntax--string .syntax--punctuation
.json .string => .syntax--json .syntax--string
.json .string .punctuation => .syntax--json .syntax--string .syntax--punctuation
.json .value > .string => .syntax--json .syntax--value > .syntax--string
.json .value > .string .punctuation => .syntax--json .syntax--value > .syntax--string .syntax--punctuation
.json .value .constant => .syntax--json .syntax--value .syntax--constant
.less.variable:first-child => .syntax--less.syntax--variable:first-child
.less .mixin => .syntax--less .syntax--mixin
.mediawiki => .syntax--mediawiki
.mediawiki .punctuation, .mediawiki .bracket, .mediawiki .brace => .syntax--mediawiki .syntax--punctuation, .syntax--mediawiki .syntax--bracket, .syntax--mediawiki .syntax--brace
.mediawiki .string .punctuation => .syntax--mediawiki .syntax--string .syntax--punctuation
.mediawiki .wiki-link => .syntax--mediawiki .syntax--wiki-link
.mediawiki .heading => .syntax--mediawiki .syntax--heading
.mediawiki .function-call => .syntax--mediawiki .syntax--function-call
.mediawiki .value => .syntax--mediawiki .syntax--value
.mediawiki .fix_this_later => .syntax--mediawiki .syntax--fix_this_later
.mediawiki .pipe, .mediawiki .link, .mediawiki .tag => .syntax--mediawiki .syntax--pipe, .syntax--mediawiki .syntax--link, .syntax--mediawiki .syntax--tag
.php .string-contents => .syntax--php .string-contents
.sass .at-rule .at-rule, .sass .at-rule .at-rule > .punctuation => .syntax--sass .syntax--at-rule .syntax--at-rule, .syntax--sass .syntax--at-rule .syntax--at-rule > .syntax--punctuation
.sass .mixin + .function, .sass .include + .function => .syntax--sass .syntax--mixin + .syntax--function, .syntax--sass .syntax--include + .syntax--function
.sass.property-value => .syntax--sass.syntax--property-value
.sass.property-value .function => .syntax--sass.syntax--property-value .syntax--function
.sass.property-value .punctuation => .syntax--sass.syntax--property-value .syntax--punctuation
.sass.variable.parameter.url => .syntax--sass.syntax--variable.syntax--parameter.syntax--url
.sass.class, .sass.parent-selector-suffix => .syntax--sass.syntax--class, .syntax--sass.syntax--parent-selector-suffix
.scss .at-rule .at-rule, .scss .at-rule .at-rule > .punctuation => .syntax--scss .syntax--at-rule .syntax--at-rule, .syntax--scss .syntax--at-rule .syntax--at-rule > .syntax--punctuation
.scss .mixin + .function, .scss .include + .function => .syntax--scss .syntax--mixin + .syntax--function, .syntax--scss .syntax--include + .syntax--function
.scss.property-value => .syntax--scss.syntax--property-value
.scss.property-value .function => .syntax--scss.syntax--property-value .syntax--function
.scss.property-value .punctuation => .syntax--scss.syntax--property-value .syntax--punctuation
.scss.variable.parameter.url => .syntax--scss.syntax--variable.syntax--parameter.syntax--url
.scss.class, .scss.parent-selector-suffix => .syntax--scss.syntax--class, .syntax--scss.syntax--parent-selector-suffix
.slim.meta => .syntax--slim.syntax--meta
.slim.text => .syntax--slim.syntax--text
.slim .string .id => .syntax--slim .syntax--string .syntax--id
.stylus .function .name => .syntax--stylus .syntax--function .syntax--name
.tex .other => .syntax--tex .syntax--other
.tex .reference => .syntax--tex .syntax--reference
.plain .text => .syntax--plain .syntax--text
.yaml .tag => .syntax--yaml .syntax--tag
.yaml .constant => .syntax--yaml .syntax--constant
.yaml .punctuation => .syntax--yaml .syntax--punctuation
```

Automatic translation of selectors will be removed in a few release cycles to minimize startup time. Please, make sure to upgrade the above selectors as soon as possible.

**Comment by simurai** [comment]
👍  Yes, should definitely be fixed before Atom `1.13` hits stable.

**Comment by varemenos** [comment]
Just giving you a gentle nudge to remind you that 1.13 hit stable
